### PR TITLE
Introduced support of org-jira

### PR DESCRIPTION
**Pull request by catap**
Introduced support of org-jira

This pull request is updated version of very old and closed https://github.com/syl20bnr/spacemacs/pull/1450
 that was updated to actual code base at both projects -- spacemacs and org-jira

### CODE REVIEW

**Comment by myrgy** [comment]
Awesome!
is it possible to add confluence support?

**Comment by catap** [comment]
@myrgy I have no idea does someone implemented `confluence-mode` or something like this.

Jira bugs/comments is much easy and this plugin has some issues with markup languge, anyway this is issue of upstream project and not of spacemacs integration.

**Comment by Compro-Prasad** [comment]
@catap Are you still on this?

**Comment by catap** [comment]
@Compro-Prasad thanks for ping. Will fix it on this week :)

p.s. honestly I forgot about this, sorry

**Comment by catap** [comment]
@Miciah thanks for review!

I've just pushed rebased and fixed version.

**Comment by catap** [comment]
rebased

File: layers/+emacs/org/README.org [hunk 1]
```diff
@@ -291,6 +293,23 @@ To permanently enable mode line display of org clock, add this snippet to your
...
+  (setq-default dotspacemacs-configuration-layers '(
+    (org :variables
+         org-enable-jira-support t
+         jiralib-url "https://yourcompany.atlassian.net:443")))
+#+END_SRC
+
+If you would like to avoid being prompted for your login and password each time you connect,
+  add your authentication credentials to =~/.authinfo.gpg= or =~/.authinfo=:
+
+#+BEGIN_SRC
```
Comment (sdwolfz) [inline layers/+emacs/org/README.org]: The build is failing because you don't have a language specified in this `BEGIN_SRC` block.

Comment (catap) [reply layers/+emacs/org/README.org]: @sdwolfz can you suggest a language for `authinfo`? I've checked https://orgmode.org/manual/Languages.html and haven't got any idea.

Comment (sdwolfz) [reply layers/+emacs/org/README.org]: I think you can safely put in `authinfo`. It does not necessarily need to be a language for which emacs has a mode available.

Comment (catap) [reply layers/+emacs/org/README.org]: @sdwolfz I see. Pushed.

**Comment by dudelson** [comment]
Is this change ready to be merged? Would love to see org-jira support in spacemacs!

**Comment by catap** [comment]
@dudelson I can easy rebase it one more time... But... ;)

**Comment by duianto** [comment]
Just update `changelog.develop` at the end of the Org section:
https://github.com/syl20bnr/spacemacs/blob/develop/CHANGELOG.develop#org-1

Stating that you've added support for `org-jira`, and list the new key bindings.

Then rebase for the last time 😄

**Pull request closed without merging**